src/components/common/RadioButtons.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="flex justify-between max-w-100">
-    <label :key="icon.id" v-for="icon in icons">
+    <label class="cursor-pointer" :key="icon.id" v-for="icon in icons">
```

Comment:
na sam koniec, atrybuty do nowej linii, dwa jeszcze przeżyje, ale trzy to już tłum :)

Reply:
Nie na sam koniec tylko pod :key, przyzwyczajajcie się do tego, potem lepiej się kod czyta